src/levels/ButtonIntroLevel.gd

```diff
@@ -11,6 +11,10 @@ var tiles := [
 ]
 
 
+func _ready() -> void:
+	$Gate.init(false)
```

Comment:
Instead of doing all this bullshit on a per-level basis, maybe just do `export var is_open := false` in the Gate object, set that to be true or false in the level scenes, then handle it in the gate's `_ready()` function?

Reply:
Absolute gigabrain strategy. Added a fix